### ISSUE TITLE
DateTime sorting fails after modifying (via AddMinutes(), etc.)

### ISSUE THREAD

**Issue by yctri** [bridgedotnet/Bridge#3307]
IOrderedEnumerable can't compare DateTimes after they've been modified with AddMinutes(), etc. It attempts to compare ticks that have been removed from DateTime instances (as a result of being modified by their adjustment methods).

### Steps To Reproduce

https://deck.net/2a202202c30951cc29fb109379f42ee0

```csharp
public class App
{
    public static void Main()
    {
        List<DateTime> times = new List<DateTime>();
        
        DateTime dt1 = DateTime.UtcNow;
        times.Add(dt1);
        DateTime dt2 = dt1.AddMinutes(-10);
        times.Add(dt2);
        
        times = times.OrderBy(dt => dt).ToList();
        
        Console.WriteLine(times[0]);
        Console.WriteLine(times[1]);
    }
}
```

### Expected Result

```js
12/12/2017 00:05:45
12/12/2017 00:15:45
```

### Actual Result

```js
System.Exception: TypeError: Cannot read property 'low' of null
    at o (https://deck.net/resources/js/bridge/bridge.min.js?16.5.0:7:119820)
    at i.n.compare (https://deck.net/resources/js/bridge/bridge.min.js?16.5.0:7:122194)
    at System.Int64.compareTo (https://deck.net/resources/js/bridge/bridge.min.js?16.5.0:7:128325)
    at Object.compare (https://deck.net/resources/js/bridge/bridge.min.js?16.5.0:7:17168)
    at Object.compare (https://deck.net/resources/js/bridge/bridge.min.js?16.5.0:7:16772)
    at ctor.<anonymous> (https://deck.net/resources/js/bridge/bridge.min.js?16.5.0:7:235941)
    at v.compare (https://deck.net/resources/js/bridge/bridge.min.js?16.5.0:7:361451)
    at https://deck.net/resources/js/bridge/bridge.min.js?16.5.0:7:360913
    at Array.sort (native)
    at https://deck.net/resources/js/bridge/bridge.min.js?16.5.0:7:360885
```

**Comment by yctri** [comment]
I think this might be because the OrderBy() clause doesn't call the DateTimes' comparison functions(?) It just does an integer comparison on DateTime.Ticks. The latter fails because of another issue: modifying a DateTime (such as with AddMinutes()) removes its Ticks member in javascript.

**Comment by fabriciomurta** [comment]
Fixed and live in Bridge 16.6.1! Thanks!
